### PR TITLE
feat(Core/CommandScript): New GM command "player learn"

### PR DESCRIPTION
**Pull request by Stoabrogga**
##### CHANGES PROPOSED:
Don't know if it's actually needed, but I created two new GM commands to learn or unlearn spells for players without targeting the player directly:
```
.player learn #playername #spell
.player unlearn #playername #spell
```
I use them on my own server for some time now and just want to share the code. If not needed this PR can be closed.

##### ISSUES ADDRESSED:
none

##### TESTS PERFORMED:
- tested build on Ubuntu 16.04 / clang 7
- tested successfully in-game

##### HOW TO TEST THE CHANGES:
Just use the commands with a GM account or via server console, e.g.:
```
.player learn Testplayer 23223
.player unlearn Testplayer 23223
```
This will learn / unlearn "Swift White Mechanostrider" for the player "Testplayer". Also try the existing commands (you need another player as target):
```
.learn 23223
.unlearn 23223
```

##### KNOWN ISSUES AND TODO LIST:
none

##### Target branch(es):
- [x] Master
 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Stoabrogga** [comment]
Pushed a new commit using a separate CommandScript like @FrancescoBorzi suggested on Discord. This could be extended with further sub-commands in the future.

File: src/server/scripts/Commands/cs_player.cpp [hunk 1]
```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-GPL2
+ * Copyright (C) 2008-2016 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
+ */
```
Comment (FrancescoBorzi) [inline src/server/scripts/Commands/cs_player.cpp]: Since this is a new file, it needs the other copyright text (agpl)

Comment (Stoabrogga) [reply src/server/scripts/Commands/cs_player.cpp]: ok, done

**Comment by Stoabrogga** [comment]
Ok, tried to remove redundant code, should be ok now and ready to be tested.

**Comment by FrancescoBorzi** [comment]
good job @Stoabrogga and @pangolp

**Comment by asfo** [comment]
Is there a way that this could work with offline players? 🤔 

Right now only work with online players so, this could be really useful with web-related commands ...

**Pull request merged**